content/Namespaces/Namespaces.q.md

```diff
@@ -0,0 +1,2010 @@
+# Namespaces
+
++++
+
+### Part 1: Sample Code Analysis
+
+:::
+
+/// type=REPL, readonly=true, filename=[UserApp.php,UserModel.php,Main.php]
+
+```php
+<?php
+namespace App;
+
+class User
+{
+    public function __construct()
+    {
+        echo "This is a user in App namespace.";
+    }
+}
+?>
+```
+
+```php
+<?php
+namespace App\Model;
+
+class User
+{
+    public function __construct()
+    {
+        echo "This is a user in App\Model namespace.";
+    }
+}
+?>
+```
+
+```php
+<?php
+require_once("./UserApp.php");
+
+$user = new App\User();
```

Comment:
@tbasilio I think this part here answers the question __"how do you access a class globally without "use"? \Foo\Bar"__. Thoughts? Thanks.

Reply:
But it doesn't answer how to access App\User from another namespace